### PR TITLE
Craychee's workwithme guide.

### PR DESCRIPTION
**Pull request by craychee**
Hello Adam!

Your former colleague Clinton sent me here to check this out. I think this is very cool so I gave it a shot. If this project is still alive, I would love in on your sweet domain.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -6,3 +6,7 @@
 	path = template
 	url = https://github.com/abloomston/workwithme.guide-template.git
 	branch = master
+[submodule "workwithme.guide-myguide"]
```
Comment (abloomston) [inline .gitmodules]: 👋 @craychee nice to meet you! I've just modified the instructions for submitting a WWM guide to be clearer in https://github.com/abloomston/workwithme.guide/pull/12, apologies for any confusion.  The way this submodule is added, your link would be https://workwithme.guide/workwithme.guide-myguide.

To change it so your link is e.g. https://workwithme.guide/craychee:

1. Remove this submodule (https://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule) to start fresh
2. ```git submodule add -b master https://github.com/craychee/workwithme.guide-myguide.git craychee```
3. stage and push changes to your `master`

Comment (craychee) [reply .gitmodules]: @abloomston 👋 Thank you for the instructions! I guess I was just mindlessly following instructions :) TIL about `deinit`.

I think I have reset properly. What a fun exercise.

**Comment by abloomston** [comment]
@craychee looks great and thanks for using the prototype! Here's your URL: http://workwithme.guide/craychee

FYI if you'd like your wwm guide to show up on the front page (which I'd love to see!) you'll need to add this to the header section of [index.md](https://github.com/craychee/workwithme.guide-myguide/blob/master/index.md):

```
categories: [profile]
```

Was that clear enough in [the template](https://raw.githubusercontent.com/abloomston/workwithme.guide-myguide/master/index.md)? Do you think I should either add a step to [the instructions](https://github.com/abloomston/workwithme.guide/blob/master/adding-your-wwm-guide.md) or perhaps change the default to be uncommented?

**Comment by craychee** [comment]
Hey there @abloomston! What a great exercise. I am encouraging all of my colleagues to do the same so expect company :) I also tweeted about it: https://twitter.com/dev_meshev/status/1118572900674494464

Your instructions were clear. I just felt a little unsure about joining you on the front page. I can do so happily, however, if you want some company :)

**Comment by abloomston** [comment]
@craychee yes please to the company :)

Also say hi to Clinton for me!

**Pull request merged**